index.html

```diff
@@ -1,12 +1,77 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Weather Report</title>
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Cute+Font&family=Righteous&display=swap');
+    </style>
 </head>
-<body>
+    <body>
+    <header>
+    <link rel="stylesheet" href="styles/index.css" />
+        <h1>Weather</h1>
+        <h1>Report</h1>
```

Comment:
You should only have a single `<h1>` tag and you shouldn't have two headings in a row like this even for other tags.  If you want them on separate lines you can use a `<br>` instead:

```suggestion
        <h1>Weather<br>Report</h1>
```